opslevel/resource_opslevel_check_alert_source_usage.go

```diff
@@ -71,8 +71,8 @@ func NewCheckAlertSourceUsageResourceModel(ctx context.Context, check opslevel.C
 	stateModel.Owner = OptionalStringValue(string(check.Owner.Team.Id))
 
 	stateModel.AlertType = types.StringValue(string(check.AlertSourceType))
-	if planModel.AlertNamePredicate != nil {
```

Comment:
This change is predicated on this [opslevel-go PR](https://github.com/OpsLevel/opslevel-go/pull/407)